src/views/sec/l10n.json

```diff
@@ -1,18 +1,124 @@
 {
-  "sec.title": "Scratch Education Collaborative",
-  "sec.intro": "Join a global network of organizations supporting creative coding",
+  "sec.title": "Scratch Education Collaborative (SEC)",
+  "sec.intro": "Participating Orgs",
   "sec.applyNow": "Apply now!",
-  "sec.applyBanner": "Applications are open now through March 1st 2021!",
-  "sec.projectsTitle": "Overview",
-  "sec.eligibilityTitle": "Eligibility",
-  "sec.applyTitle": "How do I apply?",
-  "sec.applyDeadline": "The deadline for applying to the Scratch Education Collaborative has been extended to March 1st, 2021",
+  "sec.applyBanner": "Applications are now closed. The deadline for applying to the SEC for 2021 was March 1st.",
+  "sec.projectsTitle": "Participating Orgs",
+  "sec.yearRange": "2021-2022",
+  "sec.eligibilityTitle": "Elgibility",
+  "sec.applyTitle": "Applications are now closed",
+  "sec.applyDeadline": "The deadline for applying to the SEC for 2021 was March 1st.",
   "sec.applyButton": "Click here to apply",
+  "sec.org1": "{org1Listing} - AK, USA",
+  "sec.org1LinkText": "Alaska Native Science & Engineering Program",
```

Comment:
I don't think organization names should be translated.

Reply:
Sounds good -- just changed